### ISSUE TITLE
Add Reactive Forms sample to demo page

### ISSUE THREAD

**Issue by tomgruszowski** [tinymce/tinymce-angular#4]
This is just a suggestion as it wasn't immediately obvious this control supports Reactive Forms

**Comment by fyrkant** [comment]
Sounds like a good idea!

**Comment by fyrkant** [comment]
Are there any specific examples that you think should be added to the demo/docs? I haven't built many apps with Angular so I'm not sure how most people use the reactive forms.

**Comment by fyrkant** [comment]
@tomgruszowski 
Ok I'll take a look at ngx-bootstrap!

Regarding making the private protected I guess it wouldn't be a problem but then again I never think you should have to access those properties?

**Comment by mmccoy** [comment]
@tomgruszowski 
Were you able to get this working with the ReactiveForms (formControl) module?  Did it work out of the box, or did you have to make modifications?

**Comment by tomgruszowski** [comment]
@mmccoy yes works perfectly as any other input control. As a side note I had to add the following to get styles to work.
in .angular-cli.json
```
 "assets": [
      "assets",
      "favicon.ico",
      {
        "glob": "**/*",
        "input": "../node_modules/tinymce/skins",
        "output": "./skins"
      }
    ],
```

**Comment by pbarranis** [comment]
FWIW, I found the Angular 2+ sample and had the same concern as the OP.  I didn't want to waste time banging my head against a control that doesn't work with reactive forms (or so I expected), so I googled how to make tinymce work with Angular reactive forms.  After 15-30 mins of reading various pages, I came across this issue, and only after seeing that @tomgruszowski says it works perfectly, did I actually try it.  I have no experience with TinyMCE, and reactive forms support can occasionally be spotty, so I was fully prepared to jump ship and use a different editor.

Thankfully everything worked great.  Thanks @tomgruszowski for saving me more time googling before I just gave it a shot.
